helpers/changelog/main.go

```diff
@@ -14,6 +13,12 @@ func main() {
 	}
 	defer fh.Close()
 
+	fw, err := os.Create("RELEASE_NOTES")
```

Comment:
Hmm, if someone runs this helper locally in a checkout they might end up committing this along their changes.
But on the other hand I'm happy to argue if someone runs these helpers they should know what they are doing.

Reply:
Yeah, let's try like this and see if we can figure out another way later.

Reply:
sg